PHP/PHPhomework1.php

```diff
@@ -0,0 +1,11 @@
+<?php
+  
+$x = 0;
+for ($i = 0; $i < 51; $i++) {
+    if ($i % 2 == 1) {
```

Comment:
Thank you for your advice and column about this . I changed "==" to "===". But could you tell me if I use "==", are there any trouble will happen? I think $i is absolutely integer, so there shouldn't be any trouble even I use "==". Could you tell me what kind of problem might happen if I use "==" not "==="?

Reply:
`===` faster `==` because the equality operator == coerces, or converts, the data type temporarily to see if it’s equal to the other operand, whereas === (the identity operator) doesn’t need to do any converting whatsoever and thus less work is done, which makes it faster.
Source: https://stackoverflow.com/questions/2401478/why-is-faster-than-in-php